docs/src/getting-started/deploy-and-call-with-cli.md

```diff
@@ -0,0 +1,115 @@
+# Deploy and call a Sway contract with the fuel-core CLI
+
+## 6. Spin up a fuel node
```

Comment:
I suspect we should be using `Fuel` everywhere instead of `fuel`. Except, of course, in things like `fuel-core`.

Reply:
☝️